src/ReleaseHistory.md

```diff
@@ -145,3 +145,7 @@
 ## **v1.5.47** [Driver](https://www.nuget.org/packages/Sarif.Driver/1.5.47) | [SDK](https://www.nuget.org/packages/Sarif.Sdk/1.5.47)
  * Enable converter plugins
  * Adjust RuntimeConditions enum so that 'command line parse' error is 0x1. 
+
+## **v1.6.0** [Driver](https://www.nuget.org/packages/Sarif.Driver/1.6.0) | [SDK](https://www.nuget.org/packages/Sarif.Sdk/1.6.0)
+ * Enable persistence of base64-encoded file contents via SarifLogger.
+ * Rename AnalyzeOptions.ComputeTargetsHash to ComputeFileHashes
```

Comment:
And:

* Fix bug in Semme file conversion.

Reply:
fixed

---
In reply to: [136144420](https://github.com/Microsoft/sarif-sdk/pull/612#discussion_r136144420) [](ancestors = 136144420)